src/Microsoft.SqlTools.Hosting/Hosting/Protocol/MessageDispatcher.cs

```diff
@@ -159,12 +159,26 @@ public void Stop()
                     }
                     catch (Exception ex)
                     {
-                        Logger.Error($"{requestType.MethodName} : {ex}");
-                        await requestContext.SendError(ex.Message);
+                        string exceptionMessage = FormatException(ex);
+                        Logger.Error($"{requestType.MethodName} : {exceptionMessage}");
+                        await requestContext.SendError(exceptionMessage);
                     }
                 });
         }
 
+        private string FormatException(Exception e)
+        {
+            string res = string.Empty;
+
+            while (e != null)
+            {
+                res += e.Message + Environment.NewLine;
```

Comment:
it looks like we're adding an extra newline.  Will this ever matter when we display?  e.g. unexpected linebreaks in logs or UI messages?

Reply:
updated, and for exception without inner ex, the message should be the same as the current implementation
![image](https://user-images.githubusercontent.com/21186993/223853659-21076ae1-c76b-4ec2-9e7e-cfa6266a7e51.png)